README.md

```diff
@@ -13,7 +13,7 @@
 
 This deployment follows Dex by CoreOS & Kubernetes Documentations:
 
-* [Kubernetes OIDC Doc](https://kubernetes.io/docs/admin/authentication/#option-1---oidc-authenticator)
+* [Kubernetes OIDC Doc](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#option-1-oidc-authenticator)
 * [Dex by CoreOS](https://github.com/coreos/dex)
 * [Login App](https://github.com/Flav35/loginapp)
```

Comment:
Could you also change this link to https://github.com/fydrah/loginapp ? This also has been migrated :)

Reply:
Done